### PR TITLE
chore: Updated the README escrow example.

### PR DESCRIPTION
**Pull request by Ch1n3du**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,9 +88,9 @@ Successfully updated signer to "Bob"!
 
 claim the active contract with Bob as recipient
 ```
-> cargo run --example escrow-cli claim cfb9ad1a-9621-4ed0-9d33-f0d25a37817e
+> cargo run --example escrow-cli claim ec6b54b6-238c-435c-8b1a-536831d97bc3
```
Comment (MitchTurner) [inline README.md:91]: Didn't you also need to include the index as well?

**Comment by MitchTurner** [comment]
I think you also need to update the response from `list` so it includes the `OutputId` stuff and the recipient with `Raw`.

**Comment by MitchTurner** [comment]
Also, where you do `run` for Alice, the outputs should match. It might be worth just going through the entire process and updating them all just to make sure. I'm afraid I'm gonna miss something.

File: blockfrost-http-client/src/tests.rs [hunk 2]
```diff
@@ -1,4 +1,5 @@
 use super::*;
+use crate::keys::{my_base_addr, TESTNET};
```
Comment (MitchTurner) [inline blockfrost-http-client/src/tests.rs:2]: I assume this just got left behind from something. Was this needed?

Comment (Ch1n3du) [reply blockfrost-http-client/src/tests.rs:2]: It's used on line 58 and 73

Comment (MitchTurner) [reply blockfrost-http-client/src/tests.rs:2]: Huh. Okay. Weird. I guess since they're ignored they didn't get checked by the compiler!

**Review by MitchTurner — APPROVED** [review]

**Pull request merged**